ext/land_sim_vis/plotting_utils.jl

```diff
@@ -385,6 +386,26 @@ function LandSimVis.make_diurnal_timeseries(
             label = "Model",
             color = "blue",
         )
+        # Plot diagnostics from the second model, if provided
+        if !(isnothing(diagnostics2))
```

Comment:
maybe we could also add it to the annual timeseries and just regular timeseries plots too?

For the single column runs, it feels less likely ? that we would run twice in the same script/session to get two diagnostics in memory (which we convert to a vector with "diagnostics as vectors"), but eventually when we write to a file (like we do for global runs) I could see it being useful to read in two diagnostic files and plot them on the same plot